### PR TITLE
Revert "Update Crowdin configuration file"

### PR DESCRIPTION
**Pull request by McGiverGim**
This reverts commit 5204c39af47bd2bf333458e7eb2537a0847fdb8a.

This commit was pushed by the Crowdin-GitHub integration. The new file format does not seem right, the downloaded files from Crowdin are wrong. This commit reverts it trying to make it work again but maybe it breaks the upload of the messages file to Crowdin.

It's an urgent PR, so please merge this soon to see if it works and what it breaks.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by McGiverGim** [comment]
I've managed to export the translations from Crowdin (PR here https://github.com/betaflight/betaflight-configurator/pull/3099) so this PR is not urgent anymore. We can wait without problem.

**Review by sugaarK — APPROVED** [review]

**Pull request merged**
